nifi-nar-bundles/nifi-kafka-bundle/nifi-kafka-2-6-processors/src/main/java/org/apache/nifi/processors/kafka/pubsub/KafkaProcessorUtils.java

```diff
@@ -284,6 +314,16 @@
                         + "must be set or neither must be set.")
                     .build());
             }
+
+            final String jvmJaasConfigFile = System.getProperty("java.security.auth.login.config");
```

Comment:
For clarity, it might be helpful to declare the System property name as a static variable.